cli/lib/kontena/cli/certificate/register_command.rb

```diff
@@ -22,7 +22,7 @@ def execute
     end
 
     def ask_continue
-      puts "By registering, you agree on Let's Encrypt Terms of Service: https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf"
+      puts "By registering, you agree on Let's Encrypt Terms of Service: https://letsencrypt.org/documents/2017.11.15-LE-SA-v1.2.pdf"
```

Comment:
The TOS that the server uses for the registration is https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf

Reply:
The two PDFs are identical, though.

Reply:
I grabbed the link from: https://letsencrypt.org/repository/